### PR TITLE
Use rails_autolink gem

### PR DESCRIPTION
**Pull request by rbalekai**
HYDRASIR-134 #close

### CODE REVIEW

File: lib/generators/curate/application_template.rb [hunk 1]
```diff
@@ -16,6 +16,10 @@ def yes_with_banner?(message, banner = "*" * 80)
   gem 'curate', "~> 0.6.1"
 end
 
+with_git("Adding rails_autolink") do
+  gem 'rails_autolink'
```
Comment (jcoyne) [inline lib/generators/curate/application_template.rb]: I don't think this is necessary. You've already declared the dependency in curate.gemspec. You should just have `lib/curate.rb` do `require 'rails_autolink'`

File: tasks/curate_tasks.rake [hunk 2]
```diff
@@ -54,6 +54,7 @@ task :generate do
     gem 'simplecov', group: :test, require: false
     gem 'coveralls', group: :test, require: false
     gem 'kaminari', github: 'harai/kaminari', branch: 'route_prefix_prototype'
+    gem 'rails_autolink'
```
Comment (jcoyne) [inline tasks/curate_tasks.rake]: This is also unnecessary.

File: lib/generators/curate/application_template.rb (hunk 1 above)
Comment (rbalekai) [inline lib/generators/curate/application_template.rb]: Thanks Justin

**Comment by jeremyf** [comment]
Relates to #405 
@Xiaoming Rajesh went ahead and added the auto_link gem

**Pull request merged**